Dockerfile

```diff
@@ -46,6 +55,16 @@ COPY --link frankenphp/conf.d/10-app.ini $PHP_INI_DIR/app.conf.d/
 COPY --link --chmod=755 frankenphp/docker-entrypoint.sh /usr/local/bin/docker-entrypoint
 COPY --link frankenphp/Caddyfile /etc/frankenphp/Caddyfile
 
+RUN set -eux; \
+    groupadd -g $GID $GROUP; \
+    useradd -u $UID -g $GID --no-create-home $USER; \
+    setcap CAP_NET_BIND_SERVICE=+eip /usr/local/bin/frankenphp; \
```

Comment:
`/usr/local/bin/frankenphp` already has these permissions (without inheritance). See https://github.com/php/frankenphp/blob/c14d771fdfecec91c0d3e52d0bec79d1fec1af51/Dockerfile#L121

Reply:
Hmm okay, I thought this was explicitely needed as was denoted in the documentation ["Running as Non-Root User"](https://frankenphp.dev/docs/docker/#running-as-a-non-root-user).